src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -635,6 +639,10 @@ public function testDedicatedEntityManagerNullObject()
 
     public function testEntityManagerNullObject()
     {
+        if (method_exists(ObjectManager::class, 'isUninitializedObject')) {
+            $this->markTestSkipped('This test is not applicable to doctrine/persistence 4');
```

Comment:
Those tests are application, for the same reason that the EntityValueResolver test (and with the same issue about a bad definition of the mock)